config/schema/example.json

```diff
@@ -0,0 +1,36 @@
+{
+    "RetryAttempts": 3,
+    "RetryInterval": "PT10M",
+    "Files": [
+        {
+            "FileName": "/file1/path/to/sync",
+            "Description": "Add details about the data and purpose of the synchronization",
+            "SyncDirection": "Active2Passive",
+            "SyncType": "Immediate",
+            "RetryAttempts": 1,
+            "RetryInterval": "PT10S"
+        },
+        {
+            "FileName": "/file2/path/to/sync",
+            "Description": "Add details about the data and purpose of the synchronization",
+            "SyncDirection": "Bidirectional",
+            "SyncType": "Periodic",
+            "Periodicity": "PT10S",
+            "RetryAttempts": 2,
+            "RetryInterval": "PT10M"
+        }
+    ],
+    "Directories": [
+        {
+            "DirectoryName": "/directory1/path/to/sync",
+            "Description": "Add details about the data and purpose of the synchronization",
+            "SyncDirection": "Passive2Active",
+            "SyncType": "Periodic",
+            "Periodicity": "PT1H",
+            "RetryAttempts": 1,
+            "RetryInterval": "PT10M",
+            "ExcludeFilesList": ["/files/to/ignore/for/sync"],
```

Comment:
can these by wildcards?

Reply:
I also had the same thought. But in order to make the infrastructure ready ASAP, right now the plan is to support  only the path as value for 'ExcludeFilesList', 'IncludeFilesList', 'FileName' and 'DirectoryName' , and bring the support for regex later. Also no any use case noticed from the current list of files for syncing.

In the case of bringing the wildcard support, we should define the valid wildcard and its combination and also need to validate the same. By default JSON is not supporting the validation of wildcards. So need to explore on that part. 
The same points we need to consider while parsing also.